vllm/_ipex_ops.py

```diff
@@ -43,6 +43,9 @@ def gelu_fast(out: torch.Tensor, x: torch.Tensor) -> None:
     def gelu_new(out: torch.Tensor, x: torch.Tensor) -> None:
         out.copy_(torch.nn.functional.gelu(x))
 
+    def gelu_quick(out: torch.Tensor, x: torch.Tensor) -> None:
+        out.copy_(torch.nn.functional.gelu(x))
+
```

Comment:
Not sure why `gelu_fast` and `gelu_new` are just copy of `torch.nn.functional.gelu(x)`, but I kept the same way for `gelu_quick`

Reply:
@jikunshang Could you please take a look?

Reply:
`gelu_fast` and `gelu_new` are same implementations in ipex kernel. while `gelu_quick` can not reuse. I have asked ipex team to add this new kernel. Please leave it unimplemented and I will fix here when ipex support this. Thanks for the adding!
cc @ganyi1996ppo

Reply:
Thanks for getting back to this - will do!